mkvar.py

```diff
@@ -0,0 +1,37 @@
+"""
+Takes a .txt file of variable name strings and rewrites
+them as a .py file of GPkit variable initializations
+
+$ ls
+example.txt
+$ python mkvar.py example.txt
+$ ls
+example.py example.txt
+
+- - - - - - - example.txt - - - - - - -
+foo
+bar
+- - - - - - - - - - - - - - - - - - - - 
+
+                | |
+                | |
+               \   /
+                \ /
+                 v
```

Comment:
My ASCII-art-to-actual-code ratio is 0.56, i.e. far too low. I knew I should have made that arrow bigger....